django/core/templates/base.html

```diff
@@ -1,30 +1,102 @@
+<!doctype html>
+
 {% load static %}
-<!DOCTYPE html>
+
 <html lang="en">
-<head>
-    <title>EUCLCORP</title>
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
-    <link href="https://fonts.googleapis.com/css?family=Roboto:700,900" rel="stylesheet">
-    <link rel="stylesheet" href="{% static 'css/style.css' %}">
-</head>
-<body>
-    <main role="main" id="main">
-        <h1>EUCLCORP</h1>
-        {% block main %}{% endblock %}
-    </main>
-    <footer role="contentinfo">
-        <p>
-            <a href="https://www.birmingham.ac.uk/privacy/index.aspx" target="_blank">Privacy</a> |
-            <a href="https://www.birmingham.ac.uk/legal/index.aspx" target="_blank">Legal</a> |
-            <a href="https://www.birmingham.ac.uk/university/governance/policies-regs/information/index.aspx" target="_blank">Freedom of Information</a> |
-            <a href="{% url 'general:cookies' %}">Cookies</a> |
-            <a href="https://accessibility.bear.bham.ac.uk/statements-cal.html" target="_blank">Accessibility</a>
-        </p>
-    </footer>
-
-    <!-- JavaScript -->
-    <script src="{% static 'js/cookiesmsg.js' %}"></script>
-</body>
+
+    <head>
+        <title>EUCLCORP</title>
+
+        <!--Meta-->
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <meta name="description" content="EUCLCORP website">
+        <meta name="keywords" content="eu, court, law, birmingham, university, university of birmingham, college of arts and law">
+        <meta name="author" content="University of Birmingham">
+
+        <!-- CSS -->
```

Comment:
```suggestion
```

Reply:
The CSS includes are below, in "Custom".

Reply:
The other links (Bootstrap, Google Font, Font Awesome) are also for CSS (they all include rel="stylesheet" attributes and their href includes 'css').

The CSS comment is to group all of these together. But I've removed whitespace between these to make this clearer.